[2022-09-27T20:53:20.997+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: perfinasa.colector scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-09-27T20:53:21.006+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: perfinasa.colector scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-09-27T20:53:21.007+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-27T20:53:21.007+0000] {taskinstance.py:1363} INFO - Starting attempt 5 of 5
[2022-09-27T20:53:21.008+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-27T20:53:21.021+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): colector> on 2022-08-11 00:00:00+00:00
[2022-09-27T20:53:21.025+0000] {standard_task_runner.py:54} INFO - Started process 452 to run task
[2022-09-27T20:53:21.028+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'perfinasa', 'colector', 'scheduled__2022-08-11T00:00:00+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpcefhrv7b']
[2022-09-27T20:53:21.028+0000] {standard_task_runner.py:83} INFO - Job 306: Subtask colector
[2022-09-27T20:53:21.029+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dag.py
[2022-09-27T20:53:21.438+0000] {task_command.py:384} INFO - Running <TaskInstance: perfinasa.colector scheduled__2022-08-11T00:00:00+00:00 [running]> on host 4f0e9a210a6d
[2022-09-27T20:53:21.501+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=perfinasa
AIRFLOW_CTX_TASK_ID=colector
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=5
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-11T00:00:00+00:00
[2022-09-27T20:53:21.502+0000] {tasks.py:42} INFO - Defining DEC to export information
[2022-09-27T20:53:21.508+0000] {tasks.py:357} INFO - Reading prep dataframe
[2022-09-27T20:53:21.522+0000] {tasks.py:150} INFO - Instantiating the Browser with the requested settings
[2022-09-27T20:53:24.196+0000] {tasks.py:164} INFO - Browser configured
[2022-09-27T20:53:24.196+0000] {tasks.py:287} INFO - Removing nan URL values
[2022-09-27T20:53:24.197+0000] {tasks.py:288} INFO - Requested prices before: 28
[2022-09-27T20:53:24.200+0000] {tasks.py:293} INFO - Requested prices after: 21
[2022-09-27T20:53:24.200+0000] {distinct.py:27} INFO - Add size specifications to dataframe
[2022-09-27T20:53:24.201+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks.py", line 409, in colector
    date=date,
  File "/opt/airflow/dags/tasks.py", line 427, in conditional_distincter
    config=config,
  File "/opt/airflow/dags/distinct.py", line 30, in __init__
    pd.read_csv("./resources/size.csv", sep=";", dtype=str),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 586, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 482, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 811, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/readers.py", line 1040, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/c_parser_wrapper.py", line 51, in __init__
    self._open_handles(src, kwds)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/parsers/base_parser.py", line 229, in _open_handles
    errors=kwds.get("encoding_errors", "strict"),
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: './resources/size.csv'
[2022-09-27T20:53:24.219+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=perfinasa, task_id=colector, execution_date=20220811T000000, start_date=20220927T205320, end_date=20220927T205324
[2022-09-27T20:53:24.231+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 306 for task colector ([Errno 2] No such file or directory: './resources/size.csv'; 452)
[2022-09-27T20:53:24.250+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-27T20:53:24.277+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
